docs/SNMP_Alarms.md

```diff
@@ -0,0 +1,43 @@
+# SNMP Alarms
+
+Clearwater nodes report errors over the SNMP interface when they are in an abnormal
+state, and clear them when they return to normal. They only report errors relating to
+that node - not errors relating to the whole deployment, which is the role of an external
+monitoring service.
+
+Error indications come in two forms:
+
+*   For clearly-defined errors not based on thresholds, the Clearwater node sets an
+    ITU Alarm MIB from [RFC 3877](http://tools.ietf.org/html/3877) and sends an SNMP
+    notification to a configured external SNMP manager.
+
+*   For errors based on a threshold set on a statistic (such as latency targets or
+    number of failed connections), the Clearwater node exposes that statistic over
```

Comment:
Feel free to.  I actually don't know how to do this on clearwater-readthedocs.  Is it the same as on the wiki?